pkg/antctl/raw/multicluster/create/access_token.go

```diff
@@ -83,25 +85,30 @@ func accessTokenRunE(cmd *cobra.Command, args []string) error {
 	if len(args) != 1 {
 		return fmt.Errorf("exactly one NAME is required, got %d", len(args))
 	}
-	kubeconfig, err := raw.ResolveKubeconfig(cmd)
-	if err != nil {
-		return err
-	}
-	restconfigTmpl := rest.CopyConfig(kubeconfig)
-	raw.SetupKubeconfig(restconfigTmpl)
-	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	k8sClient, err := common.NewClient(cmd)
 	if err != nil {
 		return err
 	}
 
+	var createErr error
+	createdRes := []*unstructured.Unstructured{}
+	defer func() {
+		if createErr != nil {
+			if err := common.Rollback(cmd, k8sClient, createdRes); err != nil {
+				fmt.Fprintf(cmd.OutOrStdout(), "Failed to rollback: %v\n", err)
```

Comment:
I think it's better to show the Object Namespace and Name at least, so user can rollback by themself or just ignore. Otherwise, user may have no idea about what's happened during rollback.

Reply:
The Object's name is printed in the `rollback.go`